src/components/FilterPanel/FilterPanel.jsx

```diff
@@ -0,0 +1,501 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+
+import {
+  Dialog,
+  IconButton,
+  Toolbar,
+  Box,
+  Button,
+  TextField,
+  FormControlLabel,
+  Checkbox,
+  FormControl,
+  FormLabel,
+  FormGroup,
+  Radio,
+  RadioGroup,
+  Collapse,
+  Typography,
+  InputAdornment,
+} from '@material-ui/core';
+
+import CloseIcon from '@material-ui/icons/Close';
+import SearchIcon from '@material-ui/icons/Search';
+
+const styles = {
```

Comment:
I think we should probably move a lot of these colors to `palette.js` instead of having it repeat all over. The button colors for example.

Reply:
Using palette theme colors would return undefined when running Jest/RTL tests. Making the test suite error out which is why I used hardcoded colors. Should I go forward with changing them into palette.js colors?

Reply:
I guess we can follow up on this. I also want to change some of the way we do breakpoints